op-preimage/oracle.go

```diff
@@ -60,7 +57,7 @@ type PreimageGetter func(key [32]byte) ([]byte, error)
 
 var Preimages = map[string]string{}
```

Comment:
Do we still need the preimage map?

Reply:
No need anymore. Fixed in https://github.com/ethstorage/optimism/pull/9/commits/2a0a9fae2fde90bb7107456afd0b62a8bbbf8db0